### ISSUE TITLE
Six suggestions

### ISSUE THREAD

**Issue by bignm** [kisli/vmime#40]
- On the Doxygen generator expert tab, selecting the Build topic, please check the following options to make your online documentation complete (see below list). For example, there is NO documentation at all for C++ class "fileAttachment" otherwise. I had to generate my own VMIME Doxygen documentation from the source code so that I could see all the classes, members and methods in the library.

EXTRACT_ALL
EXTRACT_PRIVATE
EXTRACT_STATIC
EXTRACT_LOCAL_CLASSES
EXTRACT_LOCAL_METHODS
EXTRACT_ANON_NSPACES
- The document "vmime-book.pdf" has a strange internal structure that makes copy & paste of the valuable example code almost impossible. Spaces surround double-colons, double less-than signs get totally disconnected and placed on different lines, etc. Please use a better PDF generator to make it more useful.
- Can you please internally document what Example 6 is doing via C++ comments, it is awfully hard to follow.
- Please add "example8.cpp" of building a message, connecting to an SMTP server and sending that message using command line options for the server, username and password.
- Please add "example9.cpp" of connecting to a POP3 server and retrieving  then parsing a message using command line options for the server, username and password.
- Please add "example10.cpp" of connecting to an IMAP server and retrieving then parsing a message using command line options for the server, username and password.

In summary, I am using VMIME v0.9.1 for a C++ website generator project, and I love it's design and structure. However, I am saddened that so much of the source code is NOT Doxygen documented, and, while the "vmime-book.pdf" helps a lot, it is far from enough to understand many of the undocumented classes, members and methods of the VMIME library.

Thanks for a great libary! Keep up the great work helping us all in the open source community providing quality free access source.

Best regards,
Tom Straub
BigNM LLC
